### PR TITLE
Removal of participation hub section from mozilla.org

### PR DESCRIPTION
**Pull request by alisha17**
Bug 1324736 - firefox os participation hub decommissioning

Removed the participation hub section from mozilla.org.

### CODE REVIEW

File: media/css/firefox/os/firefox-os-2-5.less [hunk 1]
```diff
@@ -61,8 +61,7 @@ section {
     }
 }
 .fxos-hero-space,
-.fxos-mdn,
-.fxos-hub {
+.fxos-mdn{
```
Comment (alexgibson) [inline media/css/firefox/os/firefox-os-2-5.less]: Nit space before brace `{` here and below.

File: bedrock/firefox/templates/firefox/os/index.html [hunk 2]
```diff
@@ -85,18 +85,6 @@
...
     </section>
-    <section class="fxos-hub">
-      <div class="content-wrapper">
-        <div class="hub-container">
-          <h2>{{_('Visit the Participation Hub')}}</h2>
-          <p>{{_('Want to make Firefox OS even better? Get all the resources, tools and documentation you need from the Participation Hub.')}}</p>
-          <p><a class="more" rel="external" href="https://firefoxos.mozilla.community">{{_('Check it out')}}</a></p>
-        </div>
-        <div class="hub-image">
-          {{ high_res_img('firefox/os-new/hub.png', {'width': '241', 'height': '261', 'alt':''}) }}
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/os/index.html]: Looks like you can also delete both `firefox/os-new/hub.png` and `firefox/os-new/hub-high-res.png` as they are no longer being used.

**Comment by alexgibson** [comment]
I squashed your two commits and merged to master manually in f7ceecf17d767235e850ef3849a99aa9485456b5

Thanks again, @alisha17 👍

**Comment by alisha17** [comment]
@alexgibson No worries. Happy to help! :)

**Pull request closed without merging**